xchange-blockchain/src/test/java/org/knowm/xchange/blockchain/service/account/AccountServiceTest.java

```diff
@@ -0,0 +1,167 @@
+package org.knowm.xchange.blockchain.service.account;
+
+import org.junit.Assert;
+import org.junit.Before;
+import org.junit.Test;
+import org.knowm.xchange.blockchain.BlockchainExchange;
+import org.knowm.xchange.blockchain.service.BlockchainBaseTest;
+import org.knowm.xchange.currency.Currency;
+import org.knowm.xchange.currency.CurrencyPair;
+import org.knowm.xchange.dto.account.AccountInfo;
+import org.knowm.xchange.dto.account.Fee;
+import org.knowm.xchange.dto.account.FundingRecord;
+import org.knowm.xchange.exceptions.ExchangeException;
+import org.knowm.xchange.exceptions.ExchangeSecurityException;
+import org.knowm.xchange.service.account.AccountService;
+import org.knowm.xchange.service.trade.params.HistoryParamsFundingType;
+import org.knowm.xchange.service.trade.params.TradeHistoryParams;
+
+import java.io.IOException;
+import java.math.BigDecimal;
+import java.util.List;
+import java.util.Map;
+
+import static com.github.tomakehurst.wiremock.client.WireMock.*;
+import static org.assertj.core.api.Assertions.assertThat;
+import static org.assertj.core.api.Assertions.catchThrowable;
+import static org.knowm.xchange.blockchain.service.utils.BlockchainConstants.*;
+
+public class AccountServiceTest extends BlockchainBaseTest {
```

Comment:
Please use this order for the structure of your test:

1. Before
2. Tests
3. After
4. Utility methods.